design/bytecode.md

```diff
@@ -0,0 +1,106 @@
+# Byte code
+
```

Comment:
I think it would be great to have a short "background" or "introduction" section before everything else. For example, it is super helpful to clarify in this short section that:

- SeedInterpreter will be a register-based virtual machine.
- The design of the instructions is a derived version of Lua byte code instructions.

A reference link to [The Implementation of Lua 5.0](https://www.lua.org/doc/jucs05.pdf) or [A No-Frills Introduction to Lua 5.1 VM Instructions](https://talk.pokitto.com/uploads/default/original/2X/7/716c67a0b5b1636cbc4dc1fec232ca2536cb74d1.pdf) may also help.

Reply:
Good suggestion. Will add it.